### PR TITLE
Add customer dots to notification modal

### PR DESCRIPTION
**Pull request by PatricKangasmaki**

### CODE REVIEW

File: frontend/src/components/modals/NotifyUsersModal.module.scss [hunk 1]
```diff
@@ -17,10 +17,22 @@ hr {
...
+  flex-wrap: wrap;
+  max-width: 135px;
+  margin-left: 10px;
+  margin-bottom: 15px;
+  gap: 3px;
+}
+
+.checkboxDiv {
+  margin-top: -11px;
+}
```
Comment (PatricKangasmaki) [inline frontend/src/components/modals/NotifyUsersModal.module.scss:38]: This is to align the checkbox and email with the customer dots when there are multiple rows of customers

**Comment by PatricKangasmaki** [comment]
Last push fixed console warning: Each child in a list should have a unique "key" prop.

**Review by IidaKainu — COMMENTED** [review]
Just a small change request, otherwise looks good 👍

**Comment by juhoeerola** [comment]
Seems like this only has one truly new commit (compared to master), so I'd suggest rebasing and dropping those duplicates from this pr before merging. That should also resolve the conflicts.

**Comment by PatricKangasmaki** [comment]
Added both suggestions 👍

**Review by juhoeerola — APPROVED** [review]

**Pull request merged**